schemas/manifest/schema.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "PiWebImportPluginManifestSchema",
+  "title": "PiWeb import plug-in manifest schema",
+  "description": "Describes a PiWeb import plug-in manifest.",
+  "type": "object",
+  "required": [
+    "id",
+    "title",
+    "description",
+    "provides"
+  ],
+  "additionalProperties": true,
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "title": "Schema",
+      "description": "Pointer to the schema against which this document should be validated."
+    },
+    "id": {
+      "description": "The unique identifier of the plug-in",
+      "type": "string"
+    },
+    "assemblyPath": {
+      "description": "The path to the plug-in assembly.",
+      "format": "uri-reference",
+      "pattern": ".*\\.dll$",
+      "type": "string"
+    },
+    "localesPath": {
+      "description": "A custom path to the plug-in's localizations. Default is the folder 'locales'.",
+      "format": "uri-reference",
+      "type": "string"
+    },
+    "author": {
+      "description": "The author of the plug-in",
+      "type": "string"
+    },
+    "company": {
+      "description": "The company that provided the plug-in",
+      "type": "string"
+    },
+    "contact": {
+      "description": "The contact email address of the plug-in author or company",
+      "format": "email",
+      "pattern": "^[\\w-\\\\.]+@([\\w-]+\\.)+[\\w-]{2,4}$",
+      "type": "string"
+    },
+    "homepage": {
+      "description": "The homepage with details for the plug-in",
+      "format": "uri-reference",
+      "pattern": "https?:\/\/(www\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()@:%_\\\\+.~#?&/=]*)",
+      "type": "string"
+    },
+    "issueTracker": {
+      "description": "The issue tracker for the plug-in",
+      "format": "uri-reference",
+      "pattern": "https?:\/\/(www\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()@:%_\\\\+.~#?&/=]*)",
+      "type": "string"
+    },
+    "licenseName": {
+      "description": "The licence type of the plug-in",
+      "type": "string",
+      "enum": [
```

Comment:
We should not provide a list of licenses, as this can probably never be complete.

Reply:
Yes, fixed.